### ISSUE TITLE
Closes if locking device

### ISSUE THREAD

**Issue by theblackcrow1** [xarxer/QSpot#1]
Qspot closes if locking the device. Would be very nice if it doesn't close, because usually I want to listen to music and leave the device in my pocket but without doing random actions.

**Comment by xarxer** [comment]
Hi, first of all, wow.. I didn't think anyone was using this library.
I'm actually working on a new library with more features, it should pop up in a couple of days, so stay tuned for that!

**Comment by theblackcrow1** [comment]
Other ones are crashing if I try to log in. The interface looks also very nice and is easy to use. 
Well, im excited for the new release... Will there also be one for the N900?

**Comment by theblackcrow1** [comment]
Other ones are crashing if I try to log in. The interface looks also very nice and is easy to use.
Well, im excited for the new release... Will there also be one for the N900?

**Comment by xarxer** [comment]
Other ones as in other applications?
Yeah, ideally it will run on Windows, Linux, OSX, WP, Android and iPhone!

**Comment by gitsly** [comment]
Will it be put up on github?

**Comment by xarxer** [comment]
Yes. I'm having memory issues with the native library right now. I want to fix that first so that it runs without faults.

**Comment by theblackcrow1** [comment]
Other ones as in other spotify applications.


Am 09.02.2015 um 22:03 schrieb xarxer:

**Comment by xarxer** [comment]
qtspotify is now in a repository on github.
I haven't decided yet if the pimpl-idiom is right to use with this library though.
There's a lot of work to do, so get to it @gitsly ;-)
